libimage/search.go

```diff
@@ -65,6 +65,9 @@ type SearchOptions struct {
 	// "username[:password]".  Cannot be used in combination with
 	// Username/Password.
 	Credentials string
+	// IdentityToken is used to authenticate the user and get
+	// an access token for the registry.
+	IdentityToken string `json:"identitytoken,omitempty"`
```

Comment:
Why is the JSON annotation there? I wouldn’t expect libimage structs to be passed across processes or RPCs.